src/linker/Linker.Steps/MarkSubStepsDispatcher.cs

```diff
@@ -27,7 +27,7 @@ public abstract class MarkSubStepsDispatcher : IMarkHandler
 		List<ISubStep> on_properties;
 		List<ISubStep> on_events;
 
-		protected MarkSubStepsDispatcher (IEnumerable<ISubStep> subSteps)
+		public MarkSubStepsDispatcher (IEnumerable<ISubStep> subSteps)
```

Comment:
Is this necessary for the rest of the change - I could not see how we need that.

Reply:
It was based on PR feedback in a previous iteration that used MarkSubStepsDispatcher for the serializer logic. I think it's an improvement, but I can separate it if you prefer - LMK.